x/mint/keeper/msg_server.go

```diff
@@ -0,0 +1,48 @@
+package keeper
+
+import (
+	"context"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/notional-labs/centauri/v3/x/mint/types"
+)
+
+var _ types.MsgServer = msgServer{}
+
+// NewMsgServerImpl returns an implementation of the MsgServer interface
+// for the provided Keeper.
+func NewMsgServerImpl(keeper Keeper) types.MsgServer {
+	return &msgServer{
+		Keeper: keeper,
+	}
+}
+
+type msgServer struct {
+	Keeper
+}
+
+func (ms msgServer) FundModuleAccount(goCtx context.Context, req *types.MsgFundModuleAccount) (*types.MsgFundModuleAccountResponse, error) {
```

Comment:
Could we have allowed fund module set ? Only the account in this list can fund the module account.

Reply:
Okey